Makefile

```diff
@@ -97,6 +100,10 @@ modules-docs: chpldoc
 # dependency so parallel make executions correctly build chpldoc first.
 	$(MAKE) modules-docs-only
 
+module-docs: modules-docs
```

Comment:
I'd be fine with replacing modules-docs with module-docs myself, but that's not my call.